docs/testspec/src/14_pubkey_sig.rst

```diff
@@ -395,59 +395,59 @@ described here in the following.
    |                        | #. Check that the key is valid (see PKSIG-KEY-1)                        |
    +------------------------+-------------------------------------------------------------------------+
 
-Dilithium
----------
+ML-DSA
+------
 
-The implementation is tested for correctness using the Known Answer Test vectors
-demanded by the NIST submission and provided by the reference implementation.
+The implementation is tested for correctness using Known Answer Test vectors
+generated using an implementation associated with the ACVP project.
```

Comment:
the implementation is from https://github.com/mjosaarinen/py-acvp-pqc/tree/main which has been tested against the ACVP project.

For ML-DSA, we do not use any ACVP vectors in Botan (though I just noticed that it is stated as such in the ML-DSA commit message...)